app/Http/Controllers/RegistrationController.php

```diff
@@ -60,14 +79,12 @@ public function create()
    */
   public function store(Request $request)
   {
-      // $input = $request->only('first_name', 'last_name', 'email', 'password', 'password_confirmation', 'eligibility');
-
       $this->validate($request, $this->rules, $this->messages);
       $input = $request->only('first_name', 'last_name', 'email', 'password', 'password_confirmation', 'eligibility');
 
 
-    // We don't need to save this to the db.
-    unset($input['eligibility']);
+      // We don't need to save this to the db.
+      unset($input['eligibility']);
```

Comment:
I know this was only an indentation change, but it seems like this could probably just be excluded from the `$input` array above, right?